gitpod-io/gitpod
# Web responsiveness

## Description

- Projects page grid columns breakpoints

## Related Issue(s)


## How to test

- check `/projects` and `/projects/<some-project>/configure` page and scale down browser window

## Release Notes

```release-note
Responsive grids on:
- /projects
- /projects/<some-project>/configure
```

## Documentation
